### PR TITLE
Add LIP installation script 

### PR DESCRIPTION
**Pull request by ZePedroResende**

### CODE REVIEW

**Comment by machadovilaca** [comment]
I do think it would be cool to get them using oh-my-zsh since the beginning, maybe even with git and zsh-syntax-highlighting plugins

**Comment by ZePedroResende** [comment]
@machadovilaca  i think thats a terrible idea, they need to learn bash first.
They can learn zsh on their own.

**Comment by machadovilaca** [comment]
@ZePedroResende I don't know, zsh simply gives them some more comfort and usability, mostly with autocompletes and stuff like that, I honestly don't understand why we should "hide" it from them.

**Comment by herulume** [comment]
@machadovilaca I agree with @ZePedroResende on this. Zsh and specially oh-my-zsh would make them learn "bad" habits since they don't have any bash/cli bases (most of them a least).

**Review by herulume — APPROVED** [review]

**Comment by ZePedroResende** [comment]
@machadovilaca you have to make them learn how bash works before giving them crazy shortcuts and auto-completes that they dont have acess on another system without zsh and oh-my-zsh . You dont want to leave a whole class with bad habits

**Comment by machadovilaca** [comment]
That's a non-problem, the default auto complete from zsh its almost the same as bash, without the plugins the only main difference is the tab argument navigation. And "crazy shortcuts" like gst (git status) won't, in my opinion, affect the learning method as they are self acquired knowledge and not some magic command we tell them to use.

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# install vscode
+curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+sudo install -o root -g root -m 644 microsoft.gpg /etc/apt/trusted.gpg.d/
+sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main" > /etc/apt/sources.list.d/vscode.list'
+
+sudo apt install apt-transport-https
+sudo apt update
+sudo apt install code # or code-insiders
+
+# set vscode as the default editor (https://wiki.debian.org/DebianAlternative)
```
Comment (0urobor0s) [inline install.sh:25]: Should we do this? I mean let de OS decide and the user later change it.
Think about never seing Nano

**Comment by ZePedroResende** [comment]
@machadovilaca  i dont see why you want to give a tool like that to people that are starting to learn a completly new way to interface with the computer, they dont need it and it creates bad habits

File: install.sh (hunk 1 above)
Comment (ZePedroResende) [reply install.sh:25]: I have some concerns about this too, though must LI professor give this command to students on the first/second week

**Comment by 0urobor0s** [comment]
We should only provide the basics that we do every year but now automate it a bit
We should leave user customization to the user

**Comment by machadovilaca** [comment]
@ZePedroResende I think its useful, opinions are opinions, and maybe you are right, maybe it does harm learning, but I simple can't agree as long as in all this discussion I've read 3 times "bad habits" and in neither of those times you guys have explained what those "bad habits" are.

**Comment by machadovilaca** [comment]
However I agree with @0urobor0s, and maybe showing the existence of zsh and its potential to them would be more than enough.

**Comment by ZePedroResende** [comment]
@machadovilaca  as i said before the use of so much alias leaves people that are learning not knowing what they are doing. I think its a thing to talk about for sure but not the default from the first day

**Comment by machadovilaca** [comment]
Honestly I agree with the alias problem, I just think the tab navigation and the syntax highlighting are just 2 very useful major improvements. It has advantages and disadvantages, and as long as I don't think alias are intuitive for them to just starting using them without someone showing them their existence (not sure if I was very clear here), I would still recommend zsh.

**Comment by ZePedroResende** [comment]
@machadovilaca but if you think without the tab navigation and the syntax highlights you force them to use ls and force the use of the basic navigation commands

**Comment by machadovilaca** [comment]
@ZePedroResende I understand your concerns, but I still don't think that's how you should approach this problem, forcing them to use ls and basic navigation commands, which I still use despite of being a zsh user, won't, in my opinion, improve their skill in any shell/command interpreter.

**Pull request merged**